libraries/AP_HAL/Device.h

```diff
@@ -26,13 +26,24 @@
  */
 class AP_HAL::Device {
 public:
+    enum BusType {
+        I2C,
+        SPI,
```

Comment:
Then you don't know what the field refers to. This was my first approach when I added the type in MPU9250 and MPU6000 and later I got confused about it so I added the "BUS_TYPE" prefix.